ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -484,19 +484,33 @@ private final class SqlLedger(
   )(implicit loggingContext: LoggingContext): Future[state.SubmissionResult] = {
     enqueue { offset =>
       ledgerDao
-        .storePartyEntry(
-          CurrentOffset(offset),
-          PartyLedgerEntry.AllocationAccepted(
-            Some(submissionId),
-            timeProvider.getCurrentTime,
-            PartyDetails(party, displayName, isLocal = true),
-          ),
-        )
-        .map(_ => ())(DEC)
-        .recover { case t =>
-          //recovering from the failure so the persistence stream doesn't die
-          logger.error(s"Failed to persist party $party with offset: ${offset.toApiString}", t)
-          ()
+        .getParties(Seq(party))
```

Comment:
Do you have an idea what's the perf hit of resolving the conflict by hand in this manner. I am asking because daml-hub (and others) rely on several k of parties being created in a span of first few minutes of a ledger run

Reply:
Actually running conformance test `PartyManagementServiceIT:PMAllocateOneHundred` should give us an idea

Reply:
that is a good point, thanks for raising it @mziolekda 
I did some quick measurements locally, these are the total runtimes of that particular test you mentioned int the interesting 4 combinations:
```
postgres, mutable, with query: 1113ms, 1124ms, 965ms
postgres, mutable, without query: 952ms, 1025ms, 939ms
postgres, append-only, with query: 1055ms, 1050ms, 964ms
postgres, append-only, without query: 1017ms, 951ms, 987ms
```

For my part I am comfortable with this level of barely noticeable degradation. @mziolekda @meiersi-da  do you think we should take further precaution wrt this?

(Please note after the PR is in the final state I plan to run a benchtool LR performance verification)

Reply:
This looks ok!